flow.js

```diff
@@ -96,6 +97,9 @@ class Flow {
       if(nextFlowStep == 'repair-description' && repairProblemChanged){
         delete state.data['repairProblemBestDescription']
       }
+      if( state.data.communal == 'yes' && state.step =='address') {
```

Comment:
This if-block can be deleted since the resolution was to pass the correct next steps to flow ([here](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/86/files#diff-54de87070147bd6483cb396dde9b615c82d0684d7ef8dd228e327ea5b0d36ddcR234))